numpy/core/numeric.py

```diff
@@ -2145,8 +2145,12 @@ def fromfunction(function, shape, **kwargs):
         The function is called with N parameters, where N is the rank of
         `shape`.  Each parameter represents the coordinates of the array
         varying along a specific axis.  For example, if `shape`
-        were ``(2, 2)``, then the parameters in turn be (0, 0), (0, 1),
-        (1, 0), (1, 1).
+        were ``(2, 2)``, then the parameters would in turn be
```

Comment:
Well at the very least, the intended meaning of "in turn" before and after this change is different. While what you have now might be correct, I don't think "in turn" adds any clarity at all here.